### ISSUE TITLE
Today default for date widget

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#336]
In `_defaults.json` if you put string that matches our date patterns, the CMS will automatically give you a date widget. For example:

```json
{
 "date": "1/1/2024"
}
```

However, as the months pass this day becomes quite stale and content editors have to click to bring the widget to the current day (something that is typically wanted when creating content). It would be nice if the date widget automatically picked up on a value of `"today"` and converted this to the current date. For example:

```json
{
 "date": "today"
}
```

One challenge here is that we also try to retain the date format used in your defaults when displaying (`1-1-24` would display different than `1/1/24`), so we'd have to infer the format used here. We should probably extend the `date` field widget to allow setting this via `options` in `_schema.json`:

```json
"date": {
  "type": "date",
  "options": [
    "Jan 1, 2024"
  ]
}
```

We may want to eventually add other modifiers like `yesterday`, `+ N days`, `- N days`, etc.

**Comment by jimafisk** [comment]
Or does it make more sense the other way around, having the defaults set the format (e.g. `1-1-24`) and having schema define the adjustment (e.g. `today`) in options.

**Comment by jimafisk** [comment]
This should be available in [v0.6.54](https://github.com/plentico/plenti/releases/tag/v0.6.54)

You can now use the following options in your date schema:
- `today`
- `today+N` (where N is replaced with a real number like `1`)
- `today-N` (where N is replaced with a real number like `42`)

An example `_schema.json` file might look like:
```json
{
	"my_date": {
		"type": "date",
		"options": ["today + 3"]
	}
}
```
